Docs/Debugging/Debugging with Eclipse and pyOCD.md

```diff
@@ -0,0 +1,68 @@
+#Debugging with Eclipse and pyOCD
+
+Through debugging, you can do things such as set breakpoints, set watchpoints, view registers, view disassembly, browse memory and examine the callstack. This article explains how to debug an exported mbed project with Eclipse and pyOCD. To debug a project, first, import it with mbed CLI, then create an Eclipse Makefile project and finally set up debugger configurations. You can find more detailed explanations of these steps below.
+
+##Initial setup
+1. Download the latest version of Eclipse with CDT.
+1. Install the GNU ARM Eclipse plugin.
+	1. Open Eclipse.
+	1. Click the **Help** menu item and select **Install New Software**.
+	1. In the **Work with** box, paste the install address and press Enter: [http://redmine.laoslaser.org/boards/3/topics/635](http://sourceforge.net/projects/gnuarmeclipse/files/Eclipse/updates/).
```

Comment:
Also, this link sometimes doesn't work  so we might need to link tot his issue:
http://gnuarmeclipse.github.io/blog/2016/12/02/plugins-install-issue/